evaluate/report/csv.go

```diff
@@ -121,3 +181,164 @@ func WriteEvaluationRecord(resultPath string, model model.Model, language langua
 
 	return nil
 }
+
+// loadEvaluationRecords reads and returns the evaluation records from the evaluation CSV file.
+func loadEvaluationRecords(resultPath string) (evaluationRecords EvaluationRecords, err error) {
+	evaluationFilePath := filepath.Join(resultPath, "evaluation.csv")
+	evaluationFile, err := os.Open(evaluationFilePath)
+	if err != nil {
+		return nil, pkgerrors.WithStack(err)
+	}
+	defer evaluationFile.Close()
+
+	reader := csv.NewReader(evaluationFile)
+
+	// Check if the evaluation CSV header is correct.
+	if header, err := reader.Read(); err != nil {
+		return nil, pkgerrors.Wrap(err, "file is empty")
```

Comment:
is it always empty? would maybe say `malformed`?

Reply:
Tried writing random data to the file but no error like that. Still, I changed the error message to be more generic